ingestion-edge/README.md

```diff
@@ -42,6 +43,23 @@ docker-compose logs web
 docker-compose down --timeout 0
 ```
 
+## Configuration
+
+The ingestion-edge docker container accepts these configuration options from
+environment variables:
+
+- `ROUTE_TABLE`: a JSON list of tuples mapping `uri` to PubSub topic, defaults
+  to `[]`, tuples may include an optional third element for a list of allowed
+  methods instead of `["POST","PUT"]`
+- `QUEUE_PATH`: a filesystem path to a directory where a SQLite database may be
+  created to store requests when PubSub is unavailable, defaults to `queue`
+- `MINIMUM_DISK_FREE_BYTES`: an integer indicating the threshold of free bytes
+  at `QUEUE_PATH` below which `/__heartbeat__` will fail, defaults to `0` which
```

Comment:
If this is going to be the only reason `__heartbeat__` can fail, then this is acceptable. Otherwise this should be more explicit that is is enabling a specific check that can cause `__heartbeat__` to fail.

Reply:
this is currently the only reason heartbeat will fail. i agree that this wording should be changed if/when another heartbeat check is added.